merp_base/models/res_config.py

```diff
@@ -0,0 +1,59 @@
+﻿# Copyright 2020 VentorTech OU
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0).
+
+from odoo import models, fields, api
+from odoo import http
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class StockConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    module_merp_outgoing_routing = fields.Boolean(
+        string='Outgoing Routing'
+    )
+
+    module_merp_custom_access_rights = fields.Boolean(
+        string='Custom Access Rights',
+    )
+
+    module_merp_picking_wave = fields.Boolean(
+        string='Picking Wave',
+    )
+
+    module_merp_picking_products_skip = fields.Boolean(
+        string='Smart Skip of Products',
+    )
+
+    module_merp_instant_move = fields.Boolean(
+        string='Instant Move',
+    )
+
+    module_merp_custom_logotype = fields.Boolean(
+        string='Use Custom Logo',
+    )
+
+    merp_version = fields.Char(
+        string='Ventor/mERP Version',
+        compute='_compute_merp_version',
+        store=False,
+    )
+
+    module_merp_default_locations = fields.Boolean(
+        string='Use Advanced Locations',
+    )
```

Comment:
Why we need now this? =) We merged Default locations into merp_base and now this module is not needed

Reply:
deleted